test/perf/test_performance.py

```diff
@@ -46,6 +46,8 @@ def test_perf(perf_test_configuration, request):
         # TODO add another case where we use sex as well, or both (?)
         sensitive_features_train = dataset.race_train
         sensitive_features_test = dataset.race_test
+        dataset.y_train = dataset.y_train.astype(int)
```

Comment:
Is this change relevant?

Reply:
Yes. Not sure what you mean by that, but it doesn't run otherwise

Reply:
Why does changing where the randomisation happens require this change?

Reply:
It doesn't. It is currently broken, so to get it to work I need to make this change. I want to get this particular change in asap because it actually changes the API, so I would prefer not to do multiple PRs for minimal changes like this. That's why this is included here.